lib/octokit/client.rb

```diff
@@ -325,6 +342,14 @@ def octokit_warn(*message)
 
     private
 
+    def endpoint
+      if self.class.to_s =~ /EnterpriseManagementConsoleClient/
```

Comment:
Better yet, can we just mark this method as `protected` and let that type override?